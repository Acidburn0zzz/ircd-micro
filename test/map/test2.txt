+a=A
+b=B
+c=C
+d=D
+e=E
d
-d
d
q
