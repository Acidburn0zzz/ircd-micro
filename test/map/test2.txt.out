a=A
b=B
c=C
d=D
e=E
D
a=A
b=B
c=C
e=E
bye
